invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/communities.js

```diff
@@ -0,0 +1,257 @@
+          <div
+            dangerouslySetInnerHTML={{ __html: result.metadata.description }}
+          />
+        </Card.Description>
+      </Card.Content>
+    </Card>
+  );
+}
+
+export function CommunitiesResultsItemTemplate({ result, index }) {
+  return (
+    <Item key={index}>
+      {index == 0 ? (
+        <Item.Image size="tiny" src={result.links.logo} />
+      ) : (
+        <Item.Image size="tiny" src="/static/images/placeholder.png" />
+      )}
+      <Item.Content>
+        <Item.Header href={`/communities/${result.id}`}>
+          {result.metadata.title}
+          {result.metadata.type && (
+            <span style={{ marginLeft: "5px" }}>
+              <Label size="small">{result.metadata.type}</Label>
+            </span>
+          )}
+        </Item.Header>
+        {result.metadata.website && (
+          <Item.Description href={result.metadata.website} target="_blank">
+            <a
+              dangerouslySetInnerHTML={{
+                __html: result.metadata.website,
+              }}
+            ></a>
+          </Item.Description>
+        )}
+        <Item.Meta>
+          <div
+            dangerouslySetInnerHTML={{ __html: result.metadata.description }}
+          />
+        </Item.Meta>
+      </Item.Content>
+    </Item>
+  );
+}
+
+export const CommunitiesResults = ({
+  sortOptions,
+  paginationOptions,
+  currentResultsState,
+}) => {
+  const { total } = currentResultsState.data;
+  return (
+    total && (
+      <Grid>
+        <Grid.Row>
+          <Grid.Column width={16}>
+            <Segment>
+              <Grid>
+                <Grid.Row verticalAlign="middle" className="small padding-tb-5">
+                  <Grid.Column width={4}>
+                    <Count
+                      label={() => (
+                        <>
+                          {total} {i18next.t("result(s) found")}
+                        </>
+                      )}
+                    />
+                  </Grid.Column>
+                  <Grid.Column
+                    width={12}
+                    textAlign="right"
+                    className="padding-r-5"
+                  >
+                    {sortOptions && (
+                      <Sort
+                        values={sortOptions}
+                        label={(cmp) => (
+                          <>
+                            {i18next.t("Sort by")} {cmp}
+                          </>
+                        )}
+                      />
+                    )}
+                  </Grid.Column>
+                </Grid.Row>
+                <Grid.Row>
+                  <Grid.Column>
+                    <ResultsList />
+                  </Grid.Column>
+                </Grid.Row>
+              </Grid>
+            </Segment>
+          </Grid.Column>
+        </Grid.Row>
+        <Grid.Row verticalAlign="middle">
+          <Grid.Column width={4}></Grid.Column>
+          <Grid.Column width={8} textAlign="center">
```

Comment:
do we keep the responsive design ? if yes you might want to see into responsive props of grid columns
nit: 
```suggestion
          <Grid.Column width={8} textAlign="center" floated="right">
```